README.md

```diff
@@ -39,15 +39,15 @@ kubectl apply -f ./kubemacpool.yaml
 ### Opting-in to kubemacpool service
 
 On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
```

Comment:
Please document that it is for all secondary NAD interfaces, not for the primary one.